swift/apple/FirezoneKit/Sources/FirezoneKit/Keychain/Keychain.swift

```diff
@@ -265,6 +265,31 @@ public actor Keychain {
     }
   }
 
+  private static func appBundleId() -> String {
+    var bundleId = Bundle.main.bundleIdentifier ?? "app"
```

Comment:
Hmm, isn't this required to be set? Maybe we can guard or just crash if this isn't set? I'm not sure it makes sense to continue without a valid bundle ID?

Reply:
Okay, I don't expect it to be null in an app / tunnel context.

Reply:
Kk sounds good